src/_includes/_layouts/Home.jsx

```diff
@@ -121,19 +121,26 @@ export default ({ comp, title }) => {
             redirect_url={"projects"}
           />
         </section>
-        <section className="h-[17rem] bg-primary text-negative flex flex-col justify-evenly items-center justify-center py-16 max-lg:h-fit max-lg:gap-y-16 max-lg:py-16">
-          <h1 className="lg:text-5xl text-6xl mb-12">
-            <strong>Sponsors and Partners</strong>
+        <section
+          className="bg-negative text-negative flex flex-col justify-evenly  py-16 max-lg:h-fit max-lg:gap-y-6 max-lg:py-16"
+          id="sponsors"
+        >
+          <h1 className="lg:text-5xl text-6xl mt-10 mb-2 ml-5 text-black text-left">
```

Comment:
Similar comment to the comments made in the #163. Text should be larger on mobile view. If you addressed the above comment, keep in mind that the added padding will need to be adjusted to be smaller in mobile view too.